newm-server/src/main/kotlin/io/newm/server/features/release/model/Items.kt

```diff
@@ -0,0 +1,9 @@
+package io.newm.server.features.release.model
+
+import com.google.gson.annotations.SerializedName
+
+data class Items(
```

Comment:
Rename this to `SpotifyTrackItem`

Reply:
Yes, but it's too generic for our purposes. Renaming the class here won't affect the json decoding.